packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -353,6 +431,15 @@ export function getNetworkLogo(chainId: number) {
       return '/images/ArbitrumNovaLogo.svg'
 
     default:
+      const { isArbitrum, isOrbitChain } = isNetwork(chainId)
+      if (isArbitrum) {
+        return '/images/ArbitrumOneLogo'
```

Comment:
We only check for each individual Arbitrum network one by one, so we can check everything in the default flow in case we didn't add a network, can remove it too but I guess it doesn't hurt to keep it